releasenotes/notes/bogota-manila-rome-santiago-as-fakepulsebackends-2907dec149997a27.yaml

```diff
@@ -0,0 +1,6 @@
+---
+prelude: >
+    FakeBogota, FakeManila, FakeRome, and FakeSantiago can now be used as fake backends in Pulse experiments.
+features:
+  - |
+    FakeBogota, FakeManila, FakeRome, and FakeSantiago can now be used as fake backends in Pulse experiments.
```

Comment:
```suggestion
    The ``FakeBogota``, ``FakeManila``, ``FakeRome``, and ``FakeSantiago`` fake backends which can be found
    in the ``qiskit.test.mock`` module can now be used as backends in Pulse experiments as they now include the
    a :class:`~qiskit.providers.models.PulseDefaults` created from a snapshot of the IBM Quantum's properties.
```

Reply:
Ok thank you so much for the added articulation.
Question: where can I learn about this notation you used here
>:class:`~qiskit.providers.models.PulseDefaults`

Meaning, how should I know that I need to write :class: and the "~" ?
@mtreinish